docs/installingLocal.md

```diff
@@ -1,93 +1,58 @@
+To run without spawning a shell in the container, run:
 
 ```sh
-docker run -it --rm -p 3000:3000 -e DEV=true -v .:/PrairieLearn prairielearn/prairielearn
+docker run -it --rm -p 3000:3000 \
+  -e DEV=true -v .:/PrairieLearn \
+  prairielearn/prairielearn
 ```
 
-## Running the test suite
-
-The linters and tests for the JavaScript and Python code can be run with the following commands inside the container:
+For most development, you will need access to a shell in the container to run tests, linting, etc.
+First, start a shell in the container with:
 
 ```sh
-docker run -it --rm -p 3000:3000 -w /PrairieLearn -v .:/PrairieLearn prairielearn/prairielearn /bin/bash
-
-# You can now run the following commands inside the container:
-make lint # or run "make lint-js" and "make lint-python" separately
-make test # or "make test-js" and "make test-python"
+cd PrairieLearn
+docker run -it --rm -p 3000:3000 \
+  -w /PrairieLearn -v .:/PrairieLearn \
+  prairielearn/prairielearn /bin/bash
 ```
 
-To run specific tests you first need to run `make start-support` to start the database and other services:
+This will launch a shell inside a Docker container running the PrairieLearn image, but using the current working directory for its code. If you'd rather run the command from somewhere other than the root of the repo, replace `.` with the path to the directory in `.:/PrairieLearn`.
 
-```sh
-docker run -it --rm -p 3000:3000 -w /PrairieLearn -v .:/PrairieLearn prairielearn/prairielearn /bin/bash
+To run the PrairieLearn server in development mode, use:
 
-# following commands are inside the container:
-make start-support
-cd apps/prairielearn
-yarn mocha src/tests/getHomepage.test.js
+```sh
+make dev
 ```
 
-## Working on packages
+This will start the PrairieLearn server and automatically restart it when you make changes to the JavaScript code. To stop the server, press ++ctrl+c++. To exit the container, press ++ctrl+c++ and then ++ctrl+d++.
 
-When working on something in the `packages/` directory, you'll need to rebuild the package before any changes will become visible to other packages or apps that use the package. You can build everything with `make build`, or you can run the `dev` script in a package to rebuild it automatically whenever there are changes.
+To support workspaces in local development, use `make dev-all` or `make start-all` to run both PrairieLearn and a workspace server application. For these to work, you will need to modify your Docker invocation to support external graders and workspaces. More information is on the [instructor installation page](installing.md/#support-for-external-graders-and-workspaces).
 
-```sh
-# From the root of the repository:
-yarn workspace @prairielearn/postgres run dev
+## Development
 
-# From a specific package directory, e.g. `packages/postgres`:
-yarn dev
-```
+More information on the common commands and actions you do during development can be found on the [quickstart](./quickstart.md) page.
```

Comment:
IMO we should always refer to this as the "development quickstart" page, both here and elsewhere.

Reply:
Fixed in 11571eaf4cb629f088c608b49baefe0e835027bd.